terratest/test/k8gb_weight_test.go

```diff
@@ -27,8 +27,6 @@ import (
 )
 
 func TestWeightsExistsInLocalDNSEndpoint(t *testing.T) {
-	t.Parallel()
```

Comment:
There were some random failures in this test, I had to remove parallelism to have it stable. One cluster was taking too long discovering the DNS entries of the other cluster, but I am not sure why.
Before:
```
   --- FAIL: TestWeightsExistsInLocalDNSEndpoint (337.85s)
      --- FAIL: TestWeightsExistsInLocalDNSEndpoint/with_gslb_only (245.38s)
      --- PASS: TestWeightsExistsInLocalDNSEndpoint/with_ref (92.48s)
```

After:
```
   --- PASS: TestWeightsExistsInLocalDNSEndpoint (195.55s)
      --- PASS: TestWeightsExistsInLocalDNSEndpoint/with_gslb_only (128.23s)
      --- PASS: TestWeightsExistsInLocalDNSEndpoint/with_ref (67.32s)
```